packages/back-end/src/services/auth/index.ts

```diff
@@ -248,6 +254,45 @@ export async function processJWT(
         dateCreated: new Date(),
       });
     };
+
+    if (IS_CLOUD) {
+      const build = getBuild();
+      req.gb = new GrowthBook({
+        apiHost: "https://cdn.growthbook.io",
+        clientKey: GB_SDK_ID,
+        enableDevMode: true,
+        stickyBucketService: new BrowserCookieStickyBucketService({
+          jsCookie: req.cookies,
+        }),
+        trackingCallback: () => {
+          // TODO: How to send event to Jitsu?
+        },
+        attributes: {
+          id: user.id,
+          url: req.originalUrl,
+          cloud: IS_CLOUD,
+          freeSeats: req.organization?.freeSeats,
+          discountCode: req.organization?.discountCode,
+          organizationId: req.organization?.id,
+          accountPlan: req.organization
+            ? getAccountPlan(req.organization)
+            : undefined,
+          superAdmin: user.superAdmin,
+          orgDateCreated: req.organization?.dateCreated,
+          anonymous_id: req.cookies["gb_device_id"],
+          multiOrg: IS_MULTI_ORG,
+          role: req.organization?.members.find((m) => m.id === user.id)?.role,
+          hasLicenseKey: req.organization?.licenseKey ? true : false,
+          configFile: usingFileConfig(),
+          usingSSO: usingOpenId(),
+          buildSHA: build.sha,
+          buildDate: build.date,
+          buildVersion: build.lastVersion,
+        },
```

Comment:
Should we add a new attribute for `requestSource` to differentiate between FE and BE?

Or maybe this is solved with different client ids?